### ISSUE TITLE
How do you find consensus on the best roadmap for learning something?

### ISSUE THREAD

**Issue by dfirebaugh** [knowledge-for-good/knowledgeforgood#112]
If I build a course or resources for learning a topic, what happens if someone thinks it's not a good resource?
What if they think a different "roadmap for learning" is better?

**Comment by avidrucker** [comment]
> If I build a course or resources for learning a topic, what happens if someone thinks it's not a good resource?

I think reviews and feedback can provide helpful context on the quality and nature of "learning consumables" / "learning content". That said, every person is different, with different needs and preferences. My hope is that we can, over time create reviews that take into account people's needs and preferences as well as their "overall rating" or choice of label toggles. Google and Amazon reviews come to mind as examples of "good for now but we all need better".

> What if they think a different "roadmap for learning" is better?
I encourage creators & curators to suggest new learning routes and possibilities. I think eventually we will, as communities, experts, and laypersons, be able to pave better designed "roads for learning" a variety of subjects. We have a long way to go still, and it's an exciting time in education. My hope is that we can create a collaborative, shared map of knowledge and concepts, and that it's okay to disagree on most if not all of what is in a given map. It's fine to say, for example, "the map didn't work for me, and I figured out a different way," or "I used a different map." I interested to hear what works as well as what doesn't work for people in their learning endeavors.

**Comment by avidrucker** [comment]
> What if they think a different "roadmap for learning" is better?

I encourage creators & curators to suggest new learning routes and possibilities. I think eventually we will, as communities, experts, and laypersons, be able to pave better designed "roads for learning" a variety of subjects. We have a long way to go still, and it's an exciting time in education. My hope is that we can create a collaborative, shared map of knowledge and concepts, and that it's okay to disagree on most if not all of what is in a given map. It's fine to say, for example, "the map didn't work for me, and I figured out a different way," or "I used a different map." I interested to hear what works as well as what doesn't work for people in their learning endeavors.
